### PR TITLE
Require 2.0.0.rc1 min ls-core, fix specs to match

### PR DESCRIPTION
**Pull request by andrewvc**
Since the `port` option is now obsolete, this upgrades the specs to use the port as part of the hostname with `host:port`

### CODE REVIEW

**Comment by andrewvc** [comment]
jenkins, test it

File: spec/es_spec_helper.rb [hunk 1]
```diff
@@ -26,7 +27,7 @@ def get_port
   end
 
   def get_client
-    Elasticsearch::Client.new(:host => "#{get_host}:#{get_port}")
+    Elasticsearch::Client.new(:host => "#{get_host_port}:#{get_port}")
```
Comment (untergeek) [inline spec/es_spec_helper.rb]: You don't need the `:#{get_port}` part here, do you?  I see that `get_host_port` does that already.

File: spec/integration/outputs/retry_spec.rb [hunk 2]
```diff
@@ -30,8 +30,7 @@ def mock_actions_with_response(*resp)
       "manage_template" => true,
       "index" => "logstash-2014.11.17",
       "template_overwrite" => true,
-      "hosts" => get_host(),
-      "port" => get_port(),
+      "hosts" => get_host_port(),
```
Comment (untergeek) [inline spec/integration/outputs/retry_spec.rb]: why do some calls have `()` and some not?  Should this be made more consistent (and I can see that you're following the per-file patterns).

Comment (andrewvc) [inline spec/integration/outputs/retry_spec.rb]: Good call! Interesting that the specs still pass...

Comment (andrewvc) [inline spec/integration/outputs/retry_spec.rb]: Oops, responded to the wrong thing. Hmmm, that is just historical. Does the style guide say anything here? (I personally prefer ())

File: spec/es_spec_helper.rb (hunk 1 above)
Comment (andrewvc) [inline spec/es_spec_helper.rb]: Ah thanks!

File: spec/integration/outputs/retry_spec.rb (hunk 2 above)
Comment (untergeek) [inline spec/integration/outputs/retry_spec.rb]: I'm not sure on the style guide.  @colinsurprenant @suyograo @jordansissel ?

**Comment by untergeek** [comment]
LGTM™

**Comment by talevy** [comment]
LGTM

**Pull request closed without merging**